projects/core/components/data-list/option/option.component.ts

```diff
@@ -61,7 +61,6 @@ export class TuiOptionComponent<T = unknown> implements OnDestroy {
     @Input()
     value?: T;
 
-    // TODO: 3.0 Fix dataList type when updated to Ivy and compilation drops metadata
```

Comment:
Already fixed in [the current PR](https://github.com/Tinkoff/taiga-ui/pull/1816/files#diff-d6a2eabb7cb5a90bd778bb2b977c8962ae50b8a50d426c188e8fc43b8f8961d7R75)